fe/fe-core/src/main/java/org/apache/doris/analysis/DateLiteral.java

```diff
@@ -253,6 +253,11 @@ public static DateLiteral createMinValue(Type type) throws AnalysisException {
     }
 
     private void init(String s, Type type) throws AnalysisException {
+        // if s contains micro seconds, split it and only maintain second part
+        // because BE only save a datetime with second part
+        if (s.length() > 19) {
```

Comment:
`19` is a magic number, how about define a member variable like `MAX_DATE_VALUE_LENGTH = new String("yyyy-MM-dd hh:mm:ss").length();` ?

Reply:
done